internal/kafka/client_logger.go

```diff
@@ -20,8 +20,13 @@ func NewClientLogger(cli Client, logger logger.LoggerInterface) *clientLogger {
 
 // Produce logs the message production information and then produces it
 func (c *clientLogger) Produce(message *kafka.Message) error {
+	if err := c.client.Produce(message); err != nil {
+		c.logger.Error("Kafka client: Cannot produce message", logger.String("topic", *message.TopicPartition.Topic), logger.ByteString("key", message.Key), logger.ByteString("value", message.Value), logger.Error("error", err))
```

Comment:
if error occurred we not know if the message was produce or not 
what you thing about `Kafka client error during message produce`

Reply:
Yes, I just changed it